core/drivers/crypto/caam/hal/imx_6_7/hal_clk_mx7.c

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2018-2019 NXP
+ *
+ * Brief   CAAM Clock functions.
+ */
+#include <caam_hal_clk.h>
+#include <caam_io.h>
+#include <mm/core_memprot.h>
```

Comment:
need to include platform_config.h to get `CCM_BASE`?

Reply:
it was building without but I'll add it.